src/components/document/document-toolbar.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useContext, useEffect, useState } from "react";
+import { reaction } from "mobx";
+import { clone } from "mobx-state-tree";
+import { AppConfigContext } from "../../app-config-context";
+import { DocumentModelType } from "../../models/document/document";
+import { IToolbarModel } from "../../models/stores/problem-configuration";
+import { ToolbarComponent } from "../toolbar";
+import { useStores } from "../../hooks/use-stores";
+
+interface IToolbarProps {
+  document: DocumentModelType;
+  toolbar: IToolbarModel;
+}
+
```

Comment:
I'm trying to encourage us to put a comment at the top of each component explaining what it is. In particular, the difference between a 'document toolbar' and a 'section toolbar' would be good to document.

Reply:
Fixed in https://github.com/concord-consortium/collaborative-learning/pull/2458/commits/0fe0e88bc681f100b02fc14112401cf3f0ec54b5